packages/synapse-react-client/src/components/AccessRequirementAclEditor/AccessRequirementAclEditor.test.tsx

```diff
@@ -86,62 +91,6 @@ async function setUp(
   return { ref, component, itemRows, user }
 }
 
-const confirmItem = (
```

Comment:
Moved these utilities to new file AclEditorTestUtils so all ACL editor tests can share them. Made small tweaks to make them more generic & reusable.